web/portal/client/src/entities/UsersCdr/UsersCdr.tsx

```diff
@@ -70,19 +65,18 @@ const usersCdr: EntityInterface = {
   columns,
   customActions: Actions,
   acl: {
-    ...defaultEntityBehavior.acl,
-    iden: 'kam_users_cdrs',
+    update: false,
+    create: false,
+    read: false,
```

Comment:
```suggestion
    read: true,
```
You cannot access this entity otherwise